sealer/cmd/run.go

```diff
@@ -53,7 +53,8 @@ create cluster to your baremetal server, appoint the iplist:
 	sealer run kubernetes:v1.19.8 --masters 192.168.0.2,192.168.0.3:23,192.168.0.4:24 \
 	--nodes 192.168.0.5:25,192.168.0.6:25,192.168.0.7:27 --passwd xxx
 create a cluster with custom environment variables:
-	sealer run -e DashBoardPort=8443 mydashboard:latest registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.8 --masters 3 --nodes 3
+	sealer run -e DashBoardPort=8443 mydashboard:latest registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.8 --masters 192.168.0.2,192.168.0.3,192.168.0.4 \
```

Comment:
sealer run -e DashBoardPort=8443 mydashboard:latest 。no need to set "registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.8"

Reply:
ok